### DISCUSSION TITLE
Instruct mode is loaded instead of chat mode

### DISCUSSION THREAD

**Discussion by mykeehu** [oobabooga/text-generation-webui#3389] [Q&A]
On one of my machines, when I start the program, it always starts in instruct mode instead of chat mode. The settings are good, see below. Whatever model is loaded by default, it still starts in instruct mode. I don't have this problem on the other machines, and they have the same settings. What could be the problem?

This is the content of mysettings.yaml:
```
dark_theme: false
autoload_model: true
max_new_tokens: 512
max_new_tokens_min: 1
max_new_tokens_max: 4096
seed: -1
character: None
name1: Miki
name2: Assistant
context: This is a conversation with your Assistant. It is a computer program designed
  to help you with various tasks such as answering questions, providing recommendations,
  and helping with decision making. You can ask it anything you want and it will do
  its best to give you accurate and relevant information.
greeting: ''
turn_template: ''
custom_stopping_strings: '"\n<START>","\n\n<START>"'
stop_at_newline: false
add_bos_token: true
ban_eos_token: false
skip_special_tokens: true
truncation_length: 2048
truncation_length_min: 0
truncation_length_max: 16384
mode: chat
start_with: ''
chat_style: messenger
instruction_template: None
chat-instruct_command: 'Continue the chat dialogue below. Write a single reply for
  the character "<|character|>".


  <|prompt|>'
chat_generation_attempts: 1
chat_generation_attempts_min: 1
chat_generation_attempts_max: 10
default_extensions: []
chat_default_extensions:
- gallery
preset: 'NovelAI-Storywriter better'
prompt: QA
google_translate-language string: 'hu'
```

And this is the command line of the webui.py launcher:
`CMD_FLAGS = '--chat --no-stream --threads 2 --cpu --verbose --auto-launch --extensions google_translate FPreloader --model TheBloke_MythoLogic-Mini-7B-GGML --settings C:\\oobabooga\\mysettings.yaml'`

**Comment by mykeehu** [comment]
Plus interesting fact: if I cleared the `--cpu --threads 2` options and started it that way, it now starts in chat mode. But if I switch to a different model (for example TheBloke_airoboros-l2-7B-gpt4-2.0-GGML), it switches back to instruct mode.

**Comment by flflow** [comment]
I think it's rules in text-generation-webui\models\config.yaml

**Comment by ghost** [comment]
Possibly related:

- You should warn when file name rules are hijacking the chat mode #3617
